src/main/java/com/fasterxml/jackson/databind/ser/BeanPropertyWriter.java

```diff
@@ -459,7 +470,7 @@ Object readResolve() {
             _field = null;
         }
         if (_serializer == null) {
-            _dynamicSerializers = PropertySerializerMap.emptyForProperties();
+            _dynamicSerializers = emptyForProperties();
```

Comment:
Here also, unncessary change (at least for this PR).

Reply:
done, though it was a clean way to import statically

Reply:
I prefer avoiding static imports for method calls fwtw. Maybe that should be added in style guide (or maybe it is mentioned and CONTRIBUTING.md should refer). I know IDEs can show where method comes from but for me (and I realize this is matter of preference) little bit of redundancy can be helpful.